packages/adapter/src/methods.ts

```diff
@@ -55,7 +78,7 @@ export async function addPolkadotAsset(pluginOrigin: string): Promise<void> {
   });
 }
 
-async function sendSnapMethod(request: MetamaskPolkadotRpcRequest,
+async function _sendSnapMethod(request: MetamaskPolkadotRpcRequest,
```

Comment:
why prefix?

Reply:
I thought to keep al non exported methods prefixed with `_`, but we are not doing that generally inside code so I will revert this and `_sign` method.